glambie/data/submission_cleaning_script.py

```diff
@@ -1,6 +1,6 @@
 """
-Script that can be used to make local copies of data in the glambie-submissions google bucket, check these files for any
-formatting inconsistencies (with respect to the glambie standard), and then edit the local copies to correct these
+Script that can be used to make local copies of data in the glambie2-submissions google bucket, check these files for
```

Comment:
should this comment be generalised since its either glambie-submissions or glambie2-submissions?
```suggestion
Script that can be used to make local copies of data in the glambie submissions google bucket, check these files for
```